pkg/config/config_template.yaml

```diff
@@ -430,429 +787,405 @@ api_key:
 #     username:
 #     password:
 #     token:
-
 #   - name: zookeeper
 #     polling: true
 #     template_dir: /datadog/check_configs
 #     template_url: 127.0.0.1
 #     username:
 #     password:
 
-## You can also add additional config providers by name using their default settings,
-## and pooling enabled. This list is available as an environment variable binding.
+## @param extra_config_providers - list of strings - optional
+## Add additional config providers by name using their default settings, and pooling enabled.
+## This list is available as an environment variable binding.
 #
-# extra_config_providers:
-#   - clusterchecks
+#  extra_config_providers:
+#    - clusterchecks
 
 {{ end -}}
-{{- if .Logging }}
-# Logging
-#
-# log_level: info
-# log_file: /var/log/datadog/agent.log
-
-# Set to 'true' to output logs in JSON format
-# log_format_json: false
+{{- if .Autodiscovery }}
 
-# Set to 'false' to disable logging to stdout
-# log_to_console: true
+#######################################
+## Agent Autodiscovery Configuration ##
```

Comment:
Could we merge this with the other autodiscovery section?

Not sure why this is split in two.